packages/tooling/fast-tooling-react/src/form/custom-controls/css/color-picker-component/index.ts

```diff
@@ -0,0 +1,31 @@
+import { attr, customElement } from "@microsoft/fast-element";
```

Comment:
I can't comment on the name of the file, but could you change the name of the folder from `color-picker-component` to just `color-picker`?

Reply:
Done